charts/rs-server-adgs/templates/configmap.yaml

```diff
@@ -12,306 +12,345 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# THIS FILE WAS AUTOMATICALLY CREATED FROM:
+#  - rs-server/services/adgs/config/adgs_ws_config.template.yaml
+#  - rs-server/services/adgs/config/adgs_ws_config_token_module.template.yaml
+#  - rs-server/services/adgs/config/adgs_search_config.template.yaml
+#  - rs-server/resources/sync_config_files.py
+# DON'T MODIFY IT DIRECTLY !
+
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-service-config
   namespace: {{ .Values.namespace }}
-  
 data:
-  {{ .Values.app.eodagConfigFile }}: |-
+  {{ .Values.app.eodagConfigFile }}: |
     {{- range $k, $v := .Values.app.station }}
       {{ $k }}:
         auth:
           type: TokenAuth
-          auth_uri: '{{ $v.endpoint.url }}/oauth2/token'
+          auth_uri: {{ $v.endpoint.url }}/oauth2/token
           token_type: json
           token_key: access_token
-          request_method: POST        
+          request_method: POST
+          req_data:
```

Comment:
@agrosu1978 @Padeanu is it normal that this req_data block is present in the rs-server config file but was missing from the helm charts ?